01-builder-visualization/exercises/twitter-dogs.md

```diff
@@ -0,0 +1,199 @@
+```
+<div class="CDB-infowindow-listItem CDB-infowindow-listItem--order4">
+    <h5 class="CDB-infowindow-subtitle"></h5>
+    <h4 class="CDB-infowindow-title"><a href="{{link}}" target="_blank">View Tweet</a></h4>
+```
+
+* Click `APPLY`.
+
+![pop-up click HTML](imgs/01-dog-05.png)
+
+<hr>
+
+## 3. Widgets <a name="widgets"></a>
+
+### 3. 1. Add widgets
+
+* Navigate to  the **WIDGETS** pane.
+  * You will notice that your time slider is already present.
+* Click `ADD`.
+  * In the options of the **CATEGORY** tab, check the *category_terms* widget. Make sure you are adding a widget from the `A0` layer node.
+  * In the options of the **FORMULA** tab, check `retweetcount` and `actor_followerscount`, again making sure you are adding the widget with the source as `A0`.
+* Click on `CONTINUE`.
+
+### 3. 2. Order and edit widgets
+
+* Navigate back to the **WIDGETS** pane.
+* **Click the *time widget*.**
+  - Change the title to **When are people tweeting?**.
+  - You can change the color of the widget bars now too if you'd like.
+
+* **Click the *retweetcount* widget.**
+  - Change the title to **How popular are these tweets?**
+  - Make sure the type is set to `FORMULA`.
+  - Change the **OPERATION** to `SUM(retweetcount)`
+  - Add the **SUFFIX**: ` retweets` to let your viewers know what the number they are looking at means.
+  - Widgets are only as good as the data you provide to them. You don't always want the viewers to interpret what each item means. Go ahead and add a description for the widget. In this case, I am going to explain the purpose of a retweet. Here's what I am saying:
+    - `A tweet's life cycle isn't done  after it's been sent. Someone who sees a tweet and likes it can retweet it, and share it with all of their followers too! Talk about exponential spread!`
+
+* **Click the *category_terms* widget.**
+  - Change the title to **What breed is most popular?**
+  - Make sure the type is set to `CATEGORY`.
+  - Make sure your settings for the following are correct:
+    - **AGGREGATE BY**: `category_terms`
+    - **OPERATION**: `COUNT`
+  - You can change the color of the widget bars now too if you'd like.
+
+* **Click the *actor_followerscount*  widget.**
+  - Change the title to **How many people saw these?**
+  - Make sure the type is set to `FORMULA`.
+  - Change the **OPERATION** to `SUM(actor_followerscount)`.
+  - Add the **SUFFIX**: ` people`.
+  - Add a helpful description, here is mine:
+    - `If someone sends out a tweet, all of their followers get to see it on their timelines.`
+
+* Reorder the position of widgets as you see fit.
+
+![widget-options](imgs/01-dog-06.png)
+
+<hr>
+
+## 4. Publish and share map <a name="pub"></a>
+
+* At the bottom of the main menu, click on **SHARE** at the right button corner.
+* Go to **PUBLISH** tab and click on **PUBLISH** button that is below the Map name in order to share our map.
+* After clicking on **PUBLISH**, we can select the options that we want to share our map.
```

Comment:
I would reword this - something like - After publishing, we can see from the Publish tab different options for sharing the dataset including a URL to a public map page and iframe embed code for sharing the map on an external website.